crates/polars-ops/src/chunked_array/binary/namespace.rs

```diff
@@ -46,6 +47,38 @@ pub trait BinaryNameSpaceImpl: AsBinary {
         out
     }
 
+    fn starts_with_chunked<B: AsBinary>(&self, prefix: &B) -> BooleanChunked {
```

Comment:
I don't think we need this to be generic right? 

As in, if we make this generic, this code will get compiled for every unique instantiation which leads to more binary bloat. I don't think in this case the bloat is worth it as we can always just convert to `BinaryChunked` before calling this function.